README.md

```diff
@@ -73,27 +46,27 @@ You can also define your environment variables inside a file as shown in the fol
 docker run --env-file <file-path> -ti hazelcast/hazelcast
 ```
 
-You can also define more than one VM arguments to your Hazelcast member via `JAVA_OPTS` environment variable. Please see the following example:
+As shown below, you can use `JAVA_OPTS` environment variable if you need to pass multiple VM arguments to your Hazelcast member.
 
 ```
 docker run -e JAVA_OPTS="-Xms512M -Xmx1024M" -ti hazelcast/hazelcast
 ```
 
-### Using Hazelcast Configuration File
+# Using Custom Hazelcast Configuration File
 
-In this case, you need to mount the folder that has the Hazelcast configuration file, i.e., `hazelcast.xml`. Also, while running the Docker image, you need to give the URL of `hazelcast.config` in `JAVA_OPTS` parameter. Please see the following example:
+If you need to configure with your own `hazelcast.xml` or jar files with your own domain classes, you need to mount the folder that has those files. Also, while running the Docker image, you need to give the URL of `hazelcast.config` in `JAVA_OPTS` parameter. Please see the following example:
 
```

Comment:
We don't support URLs for `hazelcast.config`, maybe better to change to "you need to give a file path for `hazelcast.config` in `JAVA_OPTS`"

Reply:
👍